x11-libs/pango/pango-1.44.7.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit gnome2-utils meson multilib-minimal toolchain-funcs xdg
+
+DESCRIPTION="Internationalized text layout and rendering library"
+HOMEPAGE="https://www.pango.org/"
+SRC_URI="http://ftp.gnome.org/pub/GNOME/sources/pango/$(ver_cut 1-2)/${P}.tar.xz"
+
+LICENSE="LGPL-2+ FTL"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
+
+# X USE flag is simply a stub until all revdeps have been adjusted to use X(+)
+IUSE="gtk-doc +introspection test +X"
```

Comment:
I don't understand. `USE=X` already existed in previous pango versions. Why is it stubbed now?

Reply:
Because `pango-1.44.x` no longer provides an option to disable X support but we have a couple of packages in portage that depend on `pango[X]`.